README.md

```diff
@@ -74,31 +74,29 @@ This project currently uses the Notification subproject from the [Alerting plugi
 4. `./gradlew integTest -Dtests.class=*RestChangePolicyActionIT` runs a single integ class
 5.  `./gradlew integTest -Dtests.class=*RestChangePolicyActionIT -Dtests.method="test missing index"` runs a single integ test method (remember to quote the test method name if it contains spaces)
 
-When launching a cluster using one of the above commands, logs are placed in `build/cluster/run node0/elasticsearch-<version>/logs`. Though the logs are teed to the console, in practices it's best to check the actual log file.
+When launching a cluster using one of the above commands, logs are placed in `build/testclusters/integTest-0/logs`. Though the logs are teed to the console, in practices it's best to check the actual log file.
 
 ### Debugging
 
-Sometimes it's useful to attach a debugger to either the Elasticsearch cluster or the integ tests to see what's going on. When running unit tests, hit **Debug** from the IDE's gutter to debug the tests.  To debug code running in an actual server, run:
+When running unit tests, hit **Debug** from the IDE's gutter to debug the tests.  
 
-```
-./gradlew integTest --debug-jvm # to start a cluster and run integ tests
-```
+Sometimes it's useful to attach a debugger to either the Elasticsearch cluster or the integ tests to see what's going on. 
 
-OR
+To debug code running in an integ test (which exercises the server from a separate JVM), run:
```

Comment:
Confused by these changes, looks like you just moved the `./gradlew -Dtest.debug integTest` section above the `./gradlew run --debug-jvm` section. Why?

Reply:
change back, sorry